package.json

```diff
@@ -22,13 +22,18 @@
 	"main": "./out/extension.js",
 	"contributes": {
 		"configuration": {
-			"title": "magic-json configuration",
+			"title": "Magic JSON",
```

Comment:
praise: I don't what this does. I suppose it's prettier this way.

Reply:
Yes, it's purely visual. All my extensions in VSCode use this style.